configmap/manual_watcher.go

```diff
@@ -40,10 +40,10 @@ func (w *ManualWatcher) Watch(name string, o Observer) {
 	defer w.m.Unlock()
 
 	if w.observers == nil {
-		w.observers = make(map[string][]Observer)
+		w.observers = make(map[string][]Observer, 1)
 	}
 
-	wl, _ := w.observers[name]
+	wl := w.observers[name]
```

Comment:
can we just delete this and do:
w.observers[name] = append(w.observers[name], o)
?

Reply:
Done.